README.md

```diff
@@ -306,6 +306,22 @@ The file containing any policy rule implementations to be used with the _evaluat
 |---------------------|-----------|---------------------------------------|
 | Kotlin script (DSL) | Evaluator | `$ORT_CONFIG_DIR/evaluator.rules.kts` |
 
+### Protecting environment variables
+
+In order to do its analyses, ORT invokes a number of external tools, such as package managers or scanners. Especially when interacting with package managers to obtain the dependencies of the analyzed project, this can lead to the execution of code in build scripts from potentially unknown sources. One possible risk in this constellation is that untrusted code could read sensitive information from environment variables used for the ORT configuration, such as database connection strings or service credentials. This is because the environment variables of a process are by default propagated to the child processes spawned by it.
```

Comment:
Nits:
- I believe lines should be hard-wrapped at column 120 also here
- "analys*i*s"
- "*A* possible risk"